src/App.js

```diff
@@ -54,6 +54,28 @@ class App extends Component {
 		})
 	}
 
+	deleteSelected = () => {
+	const { tasks, checkedTasks } = this.state;
+
+	const deletedItems = []
+	const checkedItems = []
+
+	tasks.forEach(task => {
```

Comment:
May it be an option?
```

this.setState({
	   tasks: tasks.filter(item => !checkedTasks.include(item)),
	   checkedTasks: [],
	 })
```

Reply:
I think I need to left it, cause I need to use forEach for newTasks array.